### ISSUE TITLE
support linked adapters in cutadapt

### ISSUE THREAD

**Issue by qifei9** [ewels/MultiQC#1329]
**Is your feature request related to a problem? Please describe.**

For [linked adpaters](https://cutadapt.readthedocs.io/en/latest/guide.html#linked-adapters-combined-5-and-3-adapter) in cutadapt like:
```
cutadapt -a ^ADAPTER1...ADAPTER2 -o out.fastq.gz in.fastq.gz
```

the output will report the removed sequences from 5' and 3' end separately:

```
=== First read: Adapter background ===

Sequence: GGGGGGGGGGATATGCGATCGC...GTGTCTCAACGGTATCCGCAACT; Type: linked; Length: 22+23; 5' trimmed: 1629783 times; 3' trimmed: 115720 times


No. of allowed errors:
0-9 bp: 0; 10-19 bp: 1; 20-22 bp: 2

No. of allowed errors:
0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2

Overview of removed sequences at 5' end
length  count   expect  max.err error counts
20      60597   0.0     2       0 0 60597
21      389603  0.0     2       0 231655 157948
22      1043405 0.0     2       321979 609400 112026
23      126665  0.0     2       0 67428 59237
24      9513    0.0     2       0 0 9513



Overview of removed sequences at 3' end
length  count   expect  max.err error counts
3       28945   141088.9        0       28945
4       10134   35272.2 0       10134
5       4431    8818.1  0       4431
6       3462    2204.5  0       3462
7       3057    551.1   0       3057
8       2775    137.8   0       2775
9       2650    34.4    0       2596 54
10      2768    8.6     1       2535 233
11      2582    2.2     1       2385 197
...
```

however, multiQC only report the 3' end result in the Cutadapt -> Trimmed Sequence Lengths section.

**Describe the solution you'd like**

Perhaps make 2 plots, one for the 5' end result and one for the 3' end result?

**Describe alternatives you've considered**

**Additional context**

**Comment by ewels** [comment]
Great spot @qifei9 - again agree that this should be fixed. Good work on finding these edge cases! 😅 

As with the other issue, if you have a complete example file that I can use then that would be ideal.

Phil

**Comment by ewels** [comment]
Support for linked reads has been added by @ErikDanielsson in #1469 - two plots are now shown, with trimming lengths for 5' and 3'. This will be included in the upcoming v1.11 release.

Thanks for reporting @qifei9!

Phil
